src/System.Net.Http/src/System.Net.Http.builds

```diff
@@ -12,6 +12,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TargetGroup>netcore50</TargetGroup>
     </Project>
+    <Project Include="System.Net.Http.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
The think the default is `Windows_NT`. But I've seen both patterns in the various projects.  For completeness, I put both and I think both should always be present. It doesn't seem to affect anything though.